src/main/java/org/elasticsearch/transport/local/LocalTransport.java

```diff
@@ -154,6 +154,11 @@ public BoundTransportAddress boundAddress() {
     }
 
     @Override
+    public Map<String, BoundTransportAddress> profileBoundAddresses() {
+        return null;
```

Comment:
Hrm.. not sure how I feel about null here.. what about using `Collections.EMPTY_MAP`?